third-party/rsutils/include/rsutils/string/ip-address.h

```diff
@@ -0,0 +1,58 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <rsutils/json-fwd.h>
+
+#include <string>
+#include <iosfwd>
+
+
+namespace rsutils {
+namespace string {
+
+
+// An IP address, version 4.
+// 
+// 4 decimals separated by a period:
+//     W.X.Y.Z
+//
+class ip_address
+{
+    std::string _str;
+
+public:
+    ip_address() = default;
+    ip_address( ip_address && ) = default;
+    ip_address( std::string );
+
+    ip_address & operator=( ip_address const & ) = default;
```

Comment:
If we have copying `operator=` don't we need a copy constructor as well?

Reply:
Good question, I don't remember what happened here.
Added.